.github/workflows/ci.yml

```diff
@@ -16,6 +16,11 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest]
         python-version: [3.6, 3.7, 3.8]
+        exclude:
+          - os: macos-latest
+            python-version: 3.6
+          - os: macos-latest
+            python-version: 3.7
```

Comment:
Unfortunately, list syntax for the `python-version` doesn't seem to work here, so requires some copy-paste.